examples/bzlmod/BUILD.bazel

```diff
@@ -56,15 +54,15 @@ py_binary(
 # This is used for testing mulitple versions of Python. This is
 # used only when you need to support multiple versions of Python
 # in the same project.
-# py_binary_310(
-#     name = "main_310",
-#     srcs = ["__main__.py"],
-#     main = "__main__.py",
-#     visibility = ["//:__subpackages__"],
-#     deps = [
-#         ":lib",
-#     ],
-# )
+py_binary_310(
```

Comment:
@rickeylev this is not working.  I am uncertain what I have missed that has this running python 3.9.16.

```
$ bazel run //:main_310
DEBUG: /home/clove/.cache/bazel/_bazel_clove/4d81ba2c4676ec44765d3282b1133dda/external/rules_python~override/python/extensions/python.bzl:43:10: WARNING: Not creating the toolchain from sub module, with the name python_39. The root
 module has a toolchain of the same name.
INFO: Analyzed target //:main_310 (1 packages loaded, 5 targets configured).
INFO: Found 1 target...
Target //:main_310 up-to-date:
  bazel-bin/_main_310
INFO: Elapsed time: 0.299s, Critical Path: 0.02s
INFO: 1 process: 1 internal.
INFO: Build completed successfully, 1 total action
INFO: Running command line: bazel-bin/main_310
-  -
A  1
B  2
-  -
3.9.16 (main, Jan 16 2023, 22:43:33)
[Clang 15.0.7 ]
```

Reply:
Fixed when the default toolchain was made last.